### ISSUE TITLE
Move to GitHub Actions

### ISSUE THREAD

**Issue by vweevers** [Level/community#99]
- [x] `level-concat-iterator`: https://github.com/Level/concat-iterator/pull/26, https://github.com/Level/concat-iterator/commit/390d8c7bfd90b0871b3935573440c1a336bb6c9d
- [x] `level-supports`: https://github.com/Level/supports/commit/1f0452f6cf1de33185a2288b2af85c6ae2ac230b, https://github.com/Level/supports/commit/2338ee5112138b009e6adb94a55a01c18fd28346
- [x] `abstract-leveldown`: https://github.com/Level/abstract-leveldown/pull/375, https://github.com/Level/abstract-leveldown/commit/b3331f8f452e2302e3f4e59347dce7fc846e944e
- [x] `codec`: https://github.com/Level/codec/pull/65, https://github.com/Level/codec/commit/f56349c9f625008cdab918d57ade25d8092147b8
- [x] `errors`: https://github.com/Level/errors/pull/35, https://github.com/Level/errors/commit/dcdfe8b657f622c659e3a0fa998234d0431525b9
- [x] `level-iterator-stream`: https://github.com/Level/iterator-stream/pull/69
- [x] `encoding-down`: https://github.com/Level/encoding-down/pull/99, https://github.com/Level/encoding-down/commit/82d55af12cceb8bee6411c6eb435fbe5b15a64dc
- [x] `level-js`: https://github.com/Level/level-js/pull/211, https://github.com/Level/level-js/commit/44f2eed15af6bd0a4e8cbb74f5e39ab543e58ae9, https://github.com/Level/level-js/pull/212
- [x] `memdown`: https://github.com/Level/memdown/pull/208, https://github.com/Level/memdown/commit/d80dc65710a5c715b618f042176b1da91879bf1b
- [x] `leveldown`: https://github.com/Level/leveldown/pull/764
- [x] `deferred-leveldown`: https://github.com/Level/deferred-leveldown/pull/87, https://github.com/Level/deferred-leveldown/commit/711120552de806f33ac1879f9760363091b32084
- [x] `levelup`: https://github.com/Level/levelup/pull/718, https://github.com/Level/levelup/commit/521b34c77204f3d5fb390a3e3c3e617411ac541e, https://github.com/Level/levelup/commit/f4d6a1df19f8aade086feb7a226db28839656b1c
- [x] `packager`: https://github.com/Level/packager/pull/101, https://github.com/Level/packager/commit/d151d7d67e3e72dd840a1f2a9d383431007c8aea
- [x] `level`: https://github.com/Level/level/pull/196
- [x] `rocksdb`: to be copied from `leveldown` once ready (including a successful `leveldown` release)
- [x] `level-compose`: https://github.com/Level/compose/pull/10, https://github.com/Level/compose/commit/a59494a8cf8a263304fe70d59fca60f144ba7777
- [x] `level-ws`: https://github.com/Level/level-ws/pull/124
- [x] `level-test`: https://github.com/Level/level-test/pull/143
- [x] `level-mem`: https://github.com/Level/mem/pull/63, https://github.com/Level/mem/commit/f93313e08c2296087a84933d50221ec1c9318720, https://github.com/Level/mem/commit/19e15a476962b243696968c48bed3fc533c39ed8
- [x] `level-rocksdb`: https://github.com/Level/level-rocksdb/commit/d635b371f348da92fe6614e35e28f27f45fd3b7f
- [x] `subleveldown`: https://github.com/Level/subleveldown/pull/99
- [x] `multileveldown`: https://github.com/Level/multileveldown/pull/26
- [x] `level-party`: https://github.com/Level/party/pull/31
- [x] `level-bench`: https://github.com/Level/bench/pull/26
- [x] `webpack-starter`: https://github.com/Level/webpack-starter/pull/9
- [x] `browserify-starter`: https://github.com/Level/browserify-starter/pull/5
- [x] `electron-demo`: https://github.com/Level/electron-demo/pull/117
- [x] `awesome`: https://github.com/Level/awesome/commit/ccbac95951b66709ca10b74f9a2979fcb379a1c6
- [x] `community`: https://github.com/Level/community/commit/4ccd824fca0d7ce3338ab24cb3d90b161da0bad0
- [x] ~~`leveldown-hyper` and `level-hyper`~~ (slated to be deprecated)
- [x] `leveljs.org`: https://github.com/Level/leveljs.org/commit/3bd49f291e952243ec1c8d1fcd20e3f01d5df36b
- [x] `level-ttl`

On the side:

- [x] I purchased $268 USD worth of Travis credits since January (just to keep usual builds going). We have $549 USD in OpenCollective, have yet to use that. I will submit an expense.

Cleanup tasks once the above is done:

- [x] Add `github-actions` ecosystem to `dependabot.yml` where missing
  - [x] `leveljs.org`
  - [x] `subleveldown`
  - [x] `multileveldown`
  - [x] `party`
  - [x] `bench`
  - [x] `mem`
  - [x] `browserify-starter`
  - [x] `electron-demo`
  - [x] `codec`
  - [x] `level-js`
- [x] Remove `git fetch --tags` step thanks to https://github.com/vweevers/remark-changelog/commit/c3e25b1e8deb81d3d2d36ecc55a653588f9632d6
  - [x] `level-test`
  - [x] `abstract-leveldown`
  - [x] `deferred-leveldown`
  - [x] `packager`
  - [x] `subleveldown`
  - [x] `multileveldown`
  - [x] `party`
  - [x] `mem`
- [x] Remove extraneous whitespace
  - [x] `bench`
  - [x] `browserify-starter`
  - [x] `electron-demo`
- [x] Remove Travis and AppVeyor from GitHub Apps on org
- [x] Revoke tokens of and delete the `@level-ci` machine account
- [x] Cleanup coveralls account

**Comment by vweevers** [comment]
In addition, I want to move from Coveralls to Codecov. I checked [`coverallsapp/github-action`](https://github.com/coverallsapp/github-action), it requires a github token as input and sends that to `https://coveralls.io`. Nope. In contrast, the [`codecov-action`](https://github.com/codecov/codecov-action) is tokenless for public repositories.

**Comment by ralphtheninja** [comment]
@vweevers on a roll! :smile:

**Comment by vweevers** [comment]
Just in time: https://github.blog/changelog/2021-04-19-github-actions-limit-workflow-run-or-job-concurrency/

We may need this on the repositories with browser tests, to avoid too many parallel jobs against Sauce Labs.

**Comment by cristianossd** [comment]
@vweevers, I think I can contribute with these migrations. I gonna start with `packager`

**Comment by vweevers** [comment]
Thanks everyone for your contributions so far!

I'll take `leveldown` (and later `rocksdb` by copying from `leveldown`) because that one is more involved, with its prebuilds, cross compilation in docker, having both travis and appveyor, and implicitly needing `prebuildify-ci` before `npm publish` (I want to replace this with an automated npm publish flow).

**Comment by juanpicado** [comment]
- subleveldown https://github.com/Level/subleveldown/pull/99
- multileveldown https://github.com/Level/multileveldown/pull/26
- party https://github.com/Level/party/pull/31
